js/src/behaviors/tabs.ts

```diff
@@ -0,0 +1,129 @@
+/**
+ * Tab control for assessment lens
+ * @param container - Container of tab control component
+ */
+/**
+ * Tabs behaviors
+ */
+export function initTabs() {
```

Comment:
I think we also need to consider some work to modify the page url on tab selection, which is a common pattern among our content pivots. Perhaps we can sync about it in person?

Reply:
is this something that needed to be added now or can be updated if it needs to be used?

Reply:
Hey had a quick chat about this in a sync with @jdanyow, he recommended we leave it out of  here for now and write it later if we need to.